src/js/files/components/UploadItem.js

```diff
@@ -38,6 +38,7 @@ const UploadItemTitle = styled.div`
 `;
 
 const StyledFilename = styled.span`
+    margin-right: 5px;
```

Comment:
What is this doing?

Reply:
sorry, this is no longer necessary after other changes. I've removed it